bite-size-notes/week-9-d-CKF.md

```diff
@@ -0,0 +1,136 @@
+# Introduction
+
+## Scope + Objectives
+
+These notes aim to provide a concise yet thorough introduction to
+continuous Kalman Filters (CKFs). Readers will learn the fundamentals of
+CKFs, their applications in control systems, and how they differ from
+discrete Kalman Filters. Key questions addressed include:
+-   What is a continuous Kalman Filter?
+-   How is it formulated mathematically?
+
+
+## Introduction
+
+Continuous Kalman Filters are essential in control theory and signal
+processing for estimating the state of a continuous-time dynamic system.
+They are particularly useful in applications requiring real-time state
+estimation, such as navigation systems, robotics, and autonomous
+vehicles. The continuous Kalman Filter extends the discrete Kalman
+Filter to continuous-time processes, offering more accurate and smooth
+state estimates for systems modeled by differential equations.
+
+## Continuous-Time Kalman Filter
+
+We start with the continuous time dynamical system with Gaussian noise:
+$$\dot{x}(t) = Ax(t) + Bu(t) + w(t)  \quad\quad\quad\quad w(t) \sim \mathcal{N}(0,Q)$$ 
+$$y(t) = Cx(t) + Du(t) + v(t) \quad\quad\quad\quad v(t) \sim \mathcal{N}(0,R)$$
+where, $Q$ is Process noise covariance and *R* is measurement noise
+covariance. We also assume that our measurement is not affected by the
+control, so $D = 0$.
+
+To derive the Kalman filter equations for these dynamics, we employ
+discrete-time dynamics and approximate the equations accordingly. We
+write the derivatives as finite difference terms as shown below,
+$$\\dot{x} \\approx \\frac{x\_{t+1}-x\_{t}}{\\Delta t}$$
+where $Δt → 0$.
+
```

Comment:
Note from the AMATH department:  What you are describing is using a Forward Euler discretization of a time derivative when trying to convert from continuous time to discrete time. Fwd Euler is the simplest method for discretizing derivatives.  But it is also less accurate (error is on the order of $\Delta t$), and has a limited stability region (stable only when $| 1+\Delta t \lambda | < 1$ where $\lambda$ represents the eigenvalues of the dynamics).  

This largely works for our purposes, but for alternative methods, see Finite Difference Methods for Ordinary and Partial Differential Equations: Steady-State and Time-Dependent Problems, by R Leveque (uw prof)

Reply:
I understand your concern. I did read Prof. Leveque's  book during in my undergrad CFD course. 

This is how we did in class and since at the end we are going to assume that dt is going to zero, that will eliminate the concern regarding stability.